doc/release_notes/release_notes_2.6.rst

```diff
@@ -80,8 +80,38 @@ Fixed Issues Details
 
 .. comment example item
    - :acrn-issue:`5626` - [CFL][industry] Host Call Trace once detected
-
```

Comment:
This blank line after the comment is needed

Reply:
@dbkinder Done!
Thanks!